tests/conftest.py

```diff
@@ -1,12 +1,14 @@
 from typing import List
 
+import uuid
```

Comment:
Instead of a `uuid` approach I'd recommend to extend the [`tests/utils.py:_reset_hive_org`](https://github.com/TheHive-Project/TheHive4py/blob/570066e6d6742dd014f2f06b07dec2fd2af590ae/tests/utils.py#L89C5-L89C20) method to find and delete case templates during test runs, e.g.:

```python
    [...]
    case_templates = client.case_template.find()

    with ThreadPoolExecutor() as executor:
        [...]
        executor.map(
            client.case_template.delete,
            [case_template["_id"] for case_template in case_templates],
        )
 ```
 
 This way the tests can be re-executed any number of times without relying on any unique ids.